bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/handler/RenaultHandler.java

```diff
@@ -110,7 +112,7 @@ public void initialize() {
             return;
         }
         updateStatus(ThingStatus.UNKNOWN);
-
+        updateState(CHANNEL_LOCK_STATUS, new StringType(car.getLockStatus().name()));
```

Comment:
Since `car` is not yet initialized, I'm not sure it makes sense to update the channel at this point.

Reply:
This sets the lock status to the default of "unknowen". If this lock status service works then the car will update this.  This is where I set default channel values for initialised sessions...  Maybe there is a better location?

Reply:
I think it would better to not update the channel at all at this point.

Imagine having a persisted item linked to this channel. So the first time the binding is started, item will be updated to **UNKNOWN**, then shortly after to **LOCKED** or **UNLOCKED**. After next openHAB restart, the same will happen. And you will have persisted values UNKNOWN -> LOCKED -> UNKOWN -> LOCKED.

It should only be unknown for a few milliseconds until handler is fully initialized and you have fetched the values. But actually it was also unknown prior to that, when the binding was not yet started. So what would be the advantage of updating to **UNKNOWN** for such short period of time?

Reply:
Good point. I had the problem that if the lock service got disabled the lock status was null not unknown.  I will try to handle this better.